api/v1alpha1/akodeploymentconfig_types.go

```diff
@@ -133,17 +201,23 @@ type AKOIngressConfig struct {
 	// +optional
 	DefaultIngressController bool `json:"defaultIngressController,omitempty"`
```

Comment:
ako 1.5: it's renamed to "defaultIngController"

Reply:
It is ok, we don't need to keep exactly the same name as AKO naming as long as we transferred them here: https://github.com/vmware-tanzu/community-edition/blob/main/addons/packages/load-balancer-and-ingress-service/1.4.3/bundle/config/overlays/overlay-statefulset.yaml#L42